_posts/2020-01-08-testing.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Manual Performance Testing
+date: 2020-01-08 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+Even if you have a fully unit tested app you will not be totally covered. Unit Test can only test units of code in isolation, but there is much more to an application than that: how units fit together -> Integration Tests, performance -> Performance Tests, how the user will use the app -> UI Tests + Manual Tests. Today we will take a quick look at what happened and why we had to implement a few Performance Test to understand what was going on.
```

Comment:
Performance Test - > or tests?

And what happened? What's that suddenly about?